Moderation_Matrix.md

```diff
@@ -0,0 +1,52 @@
+# Moderation Matrix
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project (Rules 11 and 12)
+-Mild unprofessionalism: see this guide https://www.bitrix24.com/about/blogs/communications/instant-messaging-and-group-chat-etiquette-in-a-professional-environme.php 
```

Comment:
In thinking about this, I am not a fan of sending users to another link, especially because several of these points are covered in our new rules under 10 (Be professional and have good etiquette). Instead, I think we should take the few things from this article not already covered and add it to our list.

Reply:
My thought is we already send people to external link in our rules to begin with.

Reply:
If we have our own list for it, it may be worth putting that link in our rules if is not fully covered.